packages/grid/_modules_/grid/context/GridContextProvider.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { GridApiContext } from '../components/GridApiContext';
+import { GridRootPropsContext } from './GridRootPropsContext';
+
+export const GridContextProvider = ({ apiRef, props, children }) => {
```

Comment:
Do we really need this wrapper? Isn't it simpler to just move the context providers to `GridComponent`?